projects/storefrontlib/src/recipes/storefront-foundation.module.ts

```diff
@@ -40,7 +40,7 @@ import { TriggerModule } from '../shared/services/trigger/trigger.module';
     UserModule.forRoot(),
     ProductModule.forRoot(),
     ViewConfigModule.forRoot(),
-    TriggerModule.forRoot(),
+    LaunchComponentModule.forRoot(),
```

Comment:
I hadn't noticed this before. Can we perhaps move the import of the `LaunchComponentModule` inside the `LayoutModule`? thoughts?